hack/verify-spelling.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# cd to the repo root
+REPO_ROOT=$(git rev-parse --show-toplevel)
+cd "${REPO_ROOT}"
+
+# create a temporary directory
+TMP_DIR=$(mktemp -d)
+
+# cleanup
+exitHandler() (
+  echo "Cleaning up ..."
+  rm -rf "${TMP_DIR}"
+)
+trap exitHandler EXIT
+
+# pull misspell
```

Comment:
why are we pulling it?

Reply:
should it be vendored - can you elaborate?

also this is still broken for me:
https://github.com/kubernetes/test-infra/blob/master/hack/verify-spelling.sh

Reply:
hmm I guess we can not vendor it, I feel like pulling will be flakier in CI. maybe what we should do is have a podutils job run an image with a misspell binary in it.

Reply:
vendoring it is worse for the `go get` users that are not developing kind 🤔

Reply:
let's go with this for now and revisit later